### PR TITLE
feat(eventhub): Org case

### PR DESCRIPTION
**Pull request by matteopasa**
Add organizational.tf to create diagnostic settings in every **Enabled** subscription of the tenant given as input.

There are 2 new input variables:
- is_organizational
- management_group_ids

is_organizational defaults to false and defines if the onboarding is of the full tenant or for a single subscription
management_group_ids defaults to empty list and defines the management groups to onboard. If the list is empty it defaults to the id of the root management group, that is the tenant ID.

### CODE REVIEW

File: modules/services/event-hub-data-source/organizational.tf [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+#---------------------------------------------------------------------------------------------
+# Create diagnostic settings for the tenant
+#---------------------------------------------------------------------------------------------
+resource "azurerm_monitor_diagnostic_setting" "sysdig_org_diagnostic_setting" {
+  count = var.is_organizational ? length(local.enabled_subscriptions) : 0
+
+  name               = "sysdig-diagnostic-setting"
+  target_resource_id = local.enabled_subscriptions[count.index].id
```
Comment (ravinadhruve10) [inline modules/services/event-hub-data-source/organizational.tf:28]: @matteopasa apologies if this was already discussed during our weekly syncups, but does this have to be done at _each subscription_ level only?

Comment (matteopasa) [reply modules/services/event-hub-data-source/organizational.tf:28]: yes unfortunately the management groups logs contain just management groups operations, we need subscription level activity logs, those contain all the logs of the resources inside the subscriptions

Comment (ravinadhruve10) [reply modules/services/event-hub-data-source/organizational.tf:28]: Ah I remember now, thanks. :)

**Review by ravinadhruve10 — APPROVED** [review]
One comment, rest lgtm.

**Pull request merged**